### PR TITLE
Package archsat.1.1

### PR DESCRIPTION
**Pull request by Gbury**
### `archsat.1.1`
A first-order theorem prover with formal proof output
Archsat is an experimental SMT/McSat solver, aimed at solving first-order problems.
Archsat integrates Tableau theory, superposition, and Rewriting into an McSat core.

Archsat currently features builtin support for equality, uninterpreted functions
and predicates, as well as rewriting. Additionally, whenever it finds a proof, it is
able to export that proof to various formal proof formats: coq proof script, coq proof term,
dedukti proof term.

---
* Homepage: https://gforge.inria.fr/projects/archsat/
* Source repo: git+https://scm.gforge.inria.fr/anonscm/git/archsat/archsat.git
* Bug tracker: https://gforge.inria.fr/tracker/?atid=14153&group_id=7473

---
:camel: Pull-request generated by opam-publish v2.0.0

### CODE REVIEW

**Comment by Gbury** [comment]
Version with updated dependency on `spelll {>= "0.3"}` (whereas `archsat 1.0` only builds with `spell {<= "0.2"}` because of the breaking switch to Seq between `0.2` and `0.3`).

File: packages/archsat/archsat.1.1/opam [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+  "qcheck"      { with-test & >= "0.8" }
+  "dolmen"      { >= "0.4" }
+  "msat"        { >= "0.7" & < "0.8" }
+  "containers"  { >= "2.0" }
+  "cmdliner"    { >= "0.9.8" }
+  "zarith"
+  "ocamlgraph"
+  "gen"
+  "mtime"
+  "sequence"    { >= "0.5" }
```
Comment (mseri) [inline packages/archsat/archsat.1.1/opam]: This package has been renamed iter (if you want to support newer versions)

**Comment by Gbury** [comment]
Right, I'll try and update this package to use `iter` (or event stdlib's Seq) instead of sequence.

**Comment by Gbury** [comment]
I've updated the PR to also switch from sequence to iter.

File: packages/archsat/archsat.1.1/opam [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    "SHAREDIR=%{archsat:share}%"
+    "install"]
+  [make
+    "-C" "static"
+    "MANDIR=%{man}%"
+    "BINDIR=%{archsat:bin}%"
+    "SHAREDIR=%{archsat:share}%"
+    "install_dk"] { dedukti:installed }
+]
+remove: [
```
Comment (kit-ty-kate) [inline packages/archsat/archsat.1.1/opam]: The `remove` section is unnecessary since opam 2.0 (except if you need to modify files during uninstall)

**Comment by kit-ty-kate** [comment]
Thanks a lot!

**Pull request merged**